sinatra-contrib.gemspec

```diff
@@ -215,7 +215,7 @@ Gem::Specification.new do |s|
     "spec/params_spec.rb",
   ]
 
-  s.add_dependency "sinatra"
+  s.add_dependency "sinatra", ">= 1.4.7"
```

Comment:
IMO this should be ~> 2.0.0, but that might not work from my experience with bundler.

What @TrevorBramble is suggesting is add a `s.required_ruby_version`, like sinatra has here:
https://github.com/sinatra/sinatra/blob/master/sinatra.gemspec#L16